### ISSUE TITLE
Add PyPNG backend

### ISSUE THREAD

**Issue by FirefoxMetzger** [imageio/imageio#749]
Another image loading library to add to our collection. This one seems relatively small and quite specific, which makes it a very beginner friendly project if anybody wants to have a go at writing a plugin :)

Name: PyPNG
Focus: PNGs (written in pure python)
GH link: https://github.com/drj11/pypng
Website: https://pypng.readthedocs.io/en/latest/index.html

If you would like to see this backend added, please react to this post or leave a comment below. The more feedback I get on a backend, the more likely I am to prioritize implementing it over solving other issues :)

**Comment by aaronepinto** [comment]
I can give this a go if it's still of interest

**Comment by FirefoxMetzger** [comment]
Yes, @aaronepinto , adding PyPNG as a backend is still interesting. Let me know if you need help getting started and I'm happy to help.
